Server.js

```diff
@@ -64,3 +64,19 @@ app.get('/cart', requiresAuth(), (req, res) => {
     user: req.oidc.user
   })
 });
+
+app.get('/products', requiresAuth(), (req,res)=>{
+
+  service.getProductsByName(req.query.name)
+    .then(function(results){
+        res.render("pages/home", {
```

Comment:
Did you connect the API to the front end? With a searchbar of some sort?

Reply:
Not sure if I know what I need to do.

Reply:
No problem.  If you look at that branch of mine that you reviewed, the PUT cart api in the Server.js is how we need the api to return the data. If you can change this /products endpoint of yours to return the data like that it'll be enough for now

Reply:
Do you have time to add it ? 
Preferably it would be nice if you can add a searchbar on the home screen where people type in a search string which triggers your api call, but if you dont have time to do that now just update the endpoint and we'll assign that task to someone else